controllers/types.go

```diff
@@ -11,3 +11,7 @@ type PorterClient interface {
 	ListInstallations(ctx context.Context, in *installationv1.ListInstallationsRequest, opts ...grpc.CallOption) (*installationv1.ListInstallationsResponse, error)
 	ListInstallationLatestOutputs(ctx context.Context, in *installationv1.ListInstallationLatestOutputRequest, opts ...grpc.CallOption) (*installationv1.ListInstallationLatestOutputResponse, error)
 }
+
+type ClientConn interface {
+	Close() error
+}
```

Comment:
Necessary to abstract to allow for unit testing. The real grpc Client will need this to close the connection.